### ISSUE TITLE
Is it possible to encrypt existing etcd volumes with kube-aws update?

### ISSUE THREAD

**Issue by mclarke47** [kubernetes-incubator/kube-aws#1347]
I have an existing cluster with unencrypted etcd volumes (0.9.9).

I am updating these cluster to the latest kube-aws version (0.10.0). As a part of this upgrade I want to encrypt the ebs volumes.
```
etcd:
  dataVolume:
    encrypted: true
```

However when I run kube-aws update I get the following cloudformation error:

```
10:47:55 UTC+0100 | UPDATE_FAILED | AWS::EC2::Volume | Etcd0EBS | Volume properties other than AutoEnableIO, type, size, and IOPS cannot be updated. Change the properties back to previous values and update the stack again.
```

Is there anyway to get around this limitation without rebuilding the cluster from scratch?

**Comment by whereisaaron** [comment]
Hi @mclarke47 AWS doesn't let you change whether a volume is encrypted or not. To encrypt the `etcd` volumes you have to delete and recreate them, something `kube-aws` actively tries to avoid for the `etcd` data volumes.

If you have 3/5/7 etcd nodes then you could manually replace each volume, making sure it retains its EIP/ENI identity (used by `kube-aws` to match the persistent data volume with the ephemeral `etcd` instance), and rely on @mumoshu's very cool automated recovery to restore that data on the new volume. Then after you finished change CloudFormation and hopefully it will accept the new state?

Bottom line, maybe, but not with `kube-aws update` and also not easily.
